DEPENDENCIES

```diff
@@ -301,33 +301,74 @@ Prerequisites for building the OpenDataPlane (ODP) API
    # ... OR directly specifying flags
    ./configure CUNIT_CFLAGS="-I/home/<my_cunit_path>/include" CUNIT_LIBS="/home/<my_cunit_path>/lib -lcunit"
 
-5.0 Documentation Images & Doxygen
+5.0 Cross Compilation for arm64
 
-  Images are stored as svg files. No conversions for these are needed.
+   To cross compile binaries for arm64 on x86_64 systems, install the following
+   required packages: crossbuild-essential-arm64, libconfig-dev:arm64,
+   libssl-dev:arm64.
```

Comment:
I guess `crossbuild-essential-arm64` package is Debian/Ubuntu specific, so I'd change the text to something like this:
```
   To cross compile binaries for arm64 on an x86_64 Debian based system, install
   the following packages: crossbuild-essential-arm64, libconfig-dev:arm64
   (optionally libssl-dev:arm64).
```